content/en/examples/admin/konnectivity/egress-selector-configuration.yaml

```diff
@@ -18,4 +18,11 @@ egressSelections:
       # The other supported transport is "tcp". You will need to set up TLS 
       # config to secure the TCP transport.
       uds:
-        udsName: /etc/srv/kubernetes/konnectivity-server/konnectivity-server.socket
+        udsName: /etc/kubernetes/konnectivity-server/konnectivity-server.socket
+
+      #tcp:
+      #  url: "https://localhost:8131"
+      #  TLSConfig:
+      #    caBundle: /etc/kubernetes/pki/konnectivity-ca.crt
+      #    clientKey: /etc/kubernetes/pki/konnectivity-client.key
+      #    clientCert: /etc/kubernetes/pki/konnectivity-client.crt
```

Comment:
These file aren't generated yet, right?

I would not worry about a TCP configuration. UDS is the cleaner way to set this up.

Reply:
> These file aren't generated yet, right?

Correct, I added this comment just in case if someone would configure HTTPConnect method instead of UDS.

I'm developing [Kubernetes-in-Kubernetes](https://github.com/kvaps/kubernetes-in-kubernetes) project and I spent some time looking for these keys since they are no where documented.

Reply:
> Correct, I added this comment just in case if someone would configure HTTPConnect method instead of UDS.

Is `HTTPConnect` a typo? I think what you meant is using `tcp` instead of `uds`.

I suggest that we either develop a full working example on how to use the tcp configuration, including how to generate the cert&key, or don't mention it at all.

Reply:
To be more clear, let's either remove this comment, or add a new section "Alternative transport configuration: TCP".

Reply:
> Is HTTPConnect a typo? I think what you meant is using tcp instead of uds.

The `HTTPConnect` is the only method able to communicate via `tcp`, isn't it?
I have an error when I try using `GRPC` with `tcp`:

```
Error: failed to validate egress selector configuration: service.connection.uds: Required value: UDS must be set with GRPC
```

> To be more clear, let's either remove this comment, or add a new section "Alternative transport configuration: TCP".

OK, I'll prepare new PR if I will have time for this.